airflow/cli/commands/info.py

```diff
@@ -0,0 +1,401 @@
+
+        return [
+            ("airflow_home", airflow_home),
+            ("system_path", os.pathsep.join(system_path)),
+            ("python_path", os.pathsep.join(python_path)),
+            ("airflow_on_path", str(airflow_on_path)),
+        ]
+
+    @property
+    def _providers_info(self):
+        return [(p.data['package-name'], p.version) for p in ProvidersManager().providers.values()]
+
+    def show(self, output: str, console: Optional[AirflowConsole] = None) -> None:
+        """Shows information about Airflow instance"""
+        all_info = {
+            "Apache Airflow": self._airflow_info,
+            "System info": self._system_info,
+            "Tools info": self._tools_info,
+            "Paths info": self._paths_info,
+            "Providers info": self._providers_info,
+        }
+
+        console = console or AirflowConsole(show_header=False)
+        if output in ("table", "plain"):
+            # Show each info as table with key, value column
+            for key, info in all_info.items():
+                console.print(f"\n[bold][green]{key}[/bold][/green]", highlight=False)
+                console.print_as(data=[{"key": k, "value": v} for k, v in info], output=output)
+        else:
+            # Render info in given format, change keys to snake_case
+            console.print_as(
+                data=[{k.lower().replace(" ", "_"): dict(v)} for k, v in all_info.items()], output=output
+            )
+
+    def render_text(self, output: str) -> str:
+        """Exports the info to string"""
+        console = AirflowConsole(record=True)
+        with console.capture():
+            self.show(output=output, console=console)
+        return console.export_text()
+
+
+class FileIoException(Exception):
+    """Raises when error happens in FileIo.io integration"""
+
+
+@tenacity.retry(
+    stop=tenacity.stop_after_attempt(5),
+    wait=tenacity.wait_exponential(multiplier=1, max=10),
+    retry=tenacity.retry_if_exception_type(FileIoException),
+    before=tenacity.before_log(log, logging.DEBUG),
+    after=tenacity.after_log(log, logging.DEBUG),
+)
+def _upload_text_to_fileio(content):
+    """Upload text file to File.io service and return lnk"""
+    resp = httpx.post("https://file.io", content=content)
+    if resp.status_code not in [200, 201]:
+        print(resp.json())
+        raise FileIoException("Failed to send report to file.io service.")
+    try:
+        return resp.json()["link"]
+    except ValueError as e:
+        log.debug(e)
+        raise FileIoException("Failed to send report to file.io service.")
+
+
+def _send_report_to_fileio(info):
+    print("Uploading report to file.io service.")
+    try:
+        link = _upload_text_to_fileio(str(info))
+        print("Report uploaded.")
+        print(link)
+        print()
+    except FileIoException as ex:
+        print(str(ex))
+
+
+@airflow_cmd.command("info")
```

Comment:
This file is a copy of `info_command.py`. The only changes (excluding import statements) are lines 376~391.